data/implementations.yaml

```diff
@@ -1271,4 +1279,4 @@
   tags:
     - rstats
     - plumber
-    - postgres
\ No newline at end of file
+    - postgres
```

Comment:
Weird that I didn't even touch this. I think the GitHub in-browser editor automatically fixes this(== no newline before EOF). You'd like me to fix this again?

Reply:
Thanks for spotting this. Don't worry about fixing it - I can resolve if if needed.